src/modules/expression-evaluator/src/evaluateExpression.ts

```diff
@@ -202,11 +215,16 @@ const extractAndSimplify = (
 }
 
 const assignChildNodesToQuery = (childNodes: any[]) => {
-  const query = childNodes[0]
-  const fieldNames = childNodes[1]
-  const values = childNodes.slice(2, fieldNames.length + 2)
-  const returnProperty = childNodes[fieldNames.length + 2]
-  return [query, fieldNames, values, returnProperty]
+  const skipFields = 3 // skip query, url and fieldNames
+  const query: string = childNodes[0]
+  const url: string = childNodes[1]
+  const fieldNames: string[] = childNodes[2]
+
+  const lastField = fieldNames.length + skipFields
```

Comment:
Perhaps `lastFieldIndex` for this variable?

Reply:
cd95ce1